runners/samza/src/main/java/org/apache/beam/runners/samza/translation/ConfigBuilder.java

```diff
@@ -90,25 +99,16 @@ private static boolean isEmptyUserConfig(Map<String, String> config) {
         .allMatch(key -> key.startsWith(SamzaRunnerOverrideConfigs.BEAM_RUNNER_CONFIG_PREFIX));
   }
 
-  private static Map<String, String> createUserConfig(SamzaPipelineOptions options)
-      throws Exception {
+  private static Map<String, String> createUserConfig(SamzaPipelineOptions options) {
     final String configFilePath = options.getConfigFilePath();
     final Map<String, String> config = new HashMap<>();
 
     // If user provides a config file, use it as base configs.
     if (StringUtils.isNoneEmpty(configFilePath)) {
       final File configFile = new File(configFilePath);
+      checkArgument(configFile.exists(), "Config file %s does not exist", configFilePath);
+      final PropertiesConfigFactory configFactory = new PropertiesConfigFactory();
```

Comment:
why forcing to use PropertiesConfigFactory here?

Reply:
Seems something got wrong in the merge. Let me rebase again with the last version.